### PR TITLE
Add location to paris-rb events

### PR DESCRIPTION
**Pull request by Spone**
... or maybe we should stick with "Paris, France" even if less accurate?

### CODE REVIEW

**Comment by adrienpoly** [comment]
Thanks, 

I would vote for Paris, not sure Villejuif really rings a bell to anyone beside french 😄 

maybe @marcoroth has an opinion on that since he added lots of location for other event.

**Comment by marcoroth** [comment]
I think I agree that Paris makes more sense, since it's also in the conference name 🙈

File: data/paris-rb/playlists.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 - id: PLjyiiigeVQV_mHcSnJnw_1LR4nzGrdF4l
   title: Paris.rb Conf 2018
+  location: "Villejuif, France"
```
Comment (Spone) [inline data/paris-rb/playlists.yml:4]:
```suggestion
  location: "Paris, France"
```

File: data/paris-rb/playlists.yml [hunk 2]
```diff
@@ -12,6 +13,7 @@
 
 - id: PLjyiiigeVQV-rqM4LD2SUbUfxXLuta1yU
   title: Paris.rb Conf 2020
+  location: "Villejuif, France"
```
Comment (Spone) [inline data/paris-rb/playlists.yml:16]:
```suggestion
  location: "Paris, France"
```

**Pull request merged**